logistics_project/apps/malawi/warehouse/report_views/resupply_qts_required.py

```diff
@@ -34,11 +36,14 @@ def custom_context(self, request):
             table["header"] = ["Facility Name"]
             facilities = facility_supply_points_below(sp.location)
         else:
-            assert is_facility(sp)
-            table["header"] = ["HSA Name"]
-            facilities = hsa_supply_points_below(sp.location)
+            if request.base_level_is_hsa:
+                assert is_facility(sp)
+                table["header"] = ["HSA Name"]
+                facilities = hsa_supply_points_below(sp.location)
+            else:
+                raise BaseLevel.InvalidReportingSupplyPointException(sp.code)
 
-        products = Product.objects.order_by('sms_code')
+        products = Product.objects.filter(type__base_level=request.base_level).order_by('sms_code')
```

Comment:
should this be filtering on active products only? it looks like inactive products are being included in this report currently